common/src/main/java/org/sonarlint/intellij/common/util/SonarLintUtils.java

```diff
@@ -96,7 +100,24 @@ public static <T> T getService(@NotNull Module module, Class<T> clazz) {
   }
 
   public static boolean isCodeScanCloudAlias(@Nullable String url) {
-    return url != null && url.contains(CODESCAN_DOMAIN);
```

Comment:
I would leave the old check in the code, no need to ping our servers - we know they are ours: 

```
if (url.contains(".codescan.io") {
  return true;
}

...otherwise new logic...

```